pywbem/_cim_types.py

```diff
@@ -655,7 +661,14 @@ def get_local_utcoffset():
         A positive value indicates minutes east of UTC, and a negative
         value indicates minutes west of UTC.
         """
-        utc = datetime.utcnow()
+        # Effective Python 3.12, utcnow must define tz to avoid warning msg
+        # That is because utcnow returns a niave datetime object if no timezone
```

Comment:
The connent suggests that utcnow() is now used with timezone, but that is not what the code does.